perun-core/src/main/java/cz/metacentrum/perun/core/impl/Utils.java

```diff
@@ -1117,11 +1117,11 @@ public static void sendSMS(String telNumber, String message) throws InternalErro
 			// handle response
 			if (exitValue == 0) {
 				// successful
-				log.info("SMS with text \"" + message + "\" to tel. number " + telNumber + " successfully sent.");
+				log.debug("SMS with text \"" + message + "\" to tel. number " + telNumber + " successfully sent.");
 			} else if ((exitValue == 1) || (exitValue == 2)) {
 				// users fault
 				String errMsg = getStringFromInputStream(process.getErrorStream());
-				log.error("Sending SMS with text \"" + message + "\" to tel. number " + telNumber + " failed.");
+				log.info("Sending SMS with text \"" + message + "\" to tel. number " + telNumber + " failed.");
```

Comment:
Please pass ``message`` and ``telNumber`` like params into info method and not as joined string with ``+``.